Packs/Rapid7AppSec/Integrations/Rapid7AppSec/Rapid7AppSec.yml

```diff
@@ -0,0 +1,738 @@
+      isArray: false
+      name: scan_type
+      predefined:
+      - Regular
+      - Verification
+      - Incremental
+      - Validation
+      required: false
+    - description: The ID of the parent scan. Relevant when scant_type=Validation/ Verification. The parent scan ID is the scan that will be updated (in case vulnerability statuses have changed) after submitting the scan. (use app-sec-scan-list to get all scan IDs).
+      isArray: false
+      name: parent_scan_id
+      required: false
+    description: Submit a new scan. A scan encapsulates all the information for a single execution of the criteria defined in the provided scan configuration.
+    execution: false
+    name: app-sec-scan-submit
+  - arguments:
+    - description: The scan ID to get the action (use app-sec-scan-list to get all scan IDs).
+      isArray: false
+      name: scan_id
+      required: true
+    description: 'Get any current scan action. Scan actions values are: "PAUSE", "RESUME", "STOP", "AUTHENTICATE", and "CANCEL". Relevant when the scan is Running and when the scan is on an action (Scan is on an action when moving between statuses. For example: After submitting a new action, the scan has an action before the new status is updated).'
+    execution: false
+    name: app-sec-scan-action-get
+    outputs:
+    - contextPath: Rapid7AppSec.Scan.id
+      description: The ID of the scan.
+      type: String
+    - contextPath: Rapid7AppSec.Scan.action
+      description: 'The scan action. Scan actions values are: "PAUSE", "RESUME", "STOP", "AUTHENTICATE", and "CANCEL".'
+      type: String
+  - arguments:
+    - description: The scan ID to submit the action (use app-sec-scan-list to get all scan IDs).
+      isArray: false
+      name: scan_id
+      required: true
+    - auto: PREDEFINED
+      description: The action to submit.
+      isArray: false
+      name: action
+      predefined:
+      - Pause
+      - Resume
+      - Stop
+      - Authenticate
+      - Cancel
+      required: true
+    - description: The interval in seconds between each poll.
+      name: interval_in_seconds
+      defaultValue: '30'
+    - description: The timeout in seconds until polling ends.
+      name: timeout_in_seconds
+      defaultValue: '600'
+    - name: first_run
+      description: First polling run.
+      hidden: true
+      defaultValue: '0'
+      required: true
+    description: "Submit a new scan action. Scan actions values are: PAUSE, RESUME, STOP, AUTHENTICATE, and CANCEL.
+     Relevant when the scan status is Running. In case the action is Stop/ Cancel, the status of the scan should be Queued/ Pending/ Running/ Provisioning."
```

Comment:
```suggestion
    description: Submit a new scan action. Scan actions values are: PAUSE, RESUME, STOP, AUTHENTICATE, and CANCEL. Relevant when the scan status is Running. In case the action is Stop/ Cancel, the status of the scan should be Queued/ Pending/ Running/ Provisioning.
```